### DISCUSSION TITLE
Is there any way to specify the third Monday of each month in CronWorkflow?

### DISCUSSION THREAD

**Discussion by panicboat** [argoproj/argo-workflows#13490] [Q&A]
### What I want to do

I want to run CronWorkflow on the 3rd Monday of every month.

### What we tried

Today, Wednesday 8/21, it worked for both of the following.

```
* * 15-21 * 4
```

```
* * 8-14 * 3
```

### Questions to ask

I understand that this is the behavior as per the [crontab](https://pubs.opengroup.org/onlinepubs/007904975/utilities/crontab.html) spec, but is there any way to achieve this?

**Comment by Joibel** [comment]
Are you saying that `* * 8-14 * 3` fires on the 21st of the Month? This seems like a bug.
Are you also saying that `* * 14-21 * 4` fires on a Wednesday, not a Thursday? This also seems wrong.

I would suggest that `x x 15-21 * 3` is the correct expression for
* Wednesday (the `3`)
* The third one of the month (`15-21`)

**Comment by panicboat** [comment]
@Joibel Thanks for confirming.
I guess I didn't explain it well.

### What we want to do

I want to schedule a CronWorkflow with the AND condition of day of the week and date by specifying the following.
```
* * 15-21 * 3
```
However, the date and day of the week seem to work under OR conditions.

#### Example

Let's say the 3rd Monday of this month is 8/19 and the current date is Wednesday 8/21.
And if we create a CronWorkflow with the following schedule, it will work in both cases, any insight on how to work around this?

##### Case1.
The day of the week does not match, but the date does
```
* * 15-21 * 4
```

##### Case2
No date match, but day of the week matches.
```
* * 8-14 * 3
 ````

**Reply by tooptoop4** [reply]
@panicboat have u tried https://github.com/argoproj/argo-workflows/pull/13474#issuecomment-2359638737
